Backend/src/routes/graph.routes.ts

```diff
@@ -0,0 +1,29 @@
+import { Router } from 'express';
+import { getGraphList, uploadGraph, removeGraph, triangleCount } from '../controllers/graph.controller';
+import multer from 'multer';
+import path from 'path';
+
+// Configure multer storage
```

Comment:
How much free storage required for this storage operation. In general how much file storage do we expect for this JasmineGraph-UI to operate?

Reply:
We need storage to accommodate graph files, with a size limit determined by the maximum upload size allowed by the frontend.

Reply:
i added a volume for temporary store the graph file in backend

Reply:
Ok